test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -210,7 +210,14 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, LossAdjustmentLifetime) {
     double timeindex[1];
     for (int i = 0; i < 1; i++)
         timeindex[i] = 100;
-    ssc_data_set_array(data, "adjust:timeindex", (ssc_number_t*)timeindex, 1);
+
+    //ssc_data_set_array(data, "adjust:timeindex", (ssc_number_t*)timeindex, 1);
+    // First, get existing adjust var table
+    var_table* adjust_vt = static_cast<var_table*>(ssc_data_get_table(data, "adjust"));
```

Comment:
It looks like that we need to add to the adjust table that we're getting from upstream, rather than create a new table? @sjanzou @brtietz Can you please check that the static cast and overall approach in this file makes sense?

Reply:
Your cast looks okay, the test is failing with the "shading" table input setup in the default case
![image](https://user-images.githubusercontent.com/6498311/218031498-02a52679-2395-401a-a056-12bd4ad817b6.png)

Reply:
@tyneises, there are many changes that need to be made upstream with updated settings for the test cases...
The "CMPvsamv1PowerIntegration_cmod_pvsamv1.LossAdjustmentLifetime" test is now passing with the latest commit using the updating of the "adjust" factor method that you were using.
![image](https://user-images.githubusercontent.com/6498311/218047354-4b928440-d465-4c06-96a9-25e15df0eab8.png)

Reply:
Thanks!